src/components/sequencing/ArgumentTooltip.svelte

```diff
@@ -5,13 +5,16 @@
   import { getAllEnumSymbols } from '../../utilities/sequence-editor/sequence-linter';
 
   export let arg: FswCommandArgument;
-  export let commandDictionary: CommandDictionary;
+  export let commandDictionary: CommandDictionary | null;
 
-  let enumSymbolsDisplayStr: string;
+  const MAX_ENUMS_TO_DISPLAY = 20;
+
+  let enumSymbolsDisplayStr: string = '';
 
   $: if (commandDictionary && arg?.arg_type === 'enum') {
     const enumValues = getAllEnumSymbols(commandDictionary.enumMap, arg.enum_name);
-    enumSymbolsDisplayStr = enumValues?.join('  |  ') ?? '';
+    const values = enumValues ?? [];
+    enumSymbolsDisplayStr = `${values.slice(0, MAX_ENUMS_TO_DISPLAY).join('  |  ')}${values.length > MAX_ENUMS_TO_DISPLAY ? '...' + (values.length - MAX_ENUMS_TO_DISPLAY) + ' more' : ''}`;
```

Comment:
Can we use template strings for all string compositions in here? It gets a little difficult to grok when it's combined with real math.

Reply:
addressed in https://github.com/NASA-AMMOS/aerie-ui/pull/1472/commits/9b825534e6b51e9c9953a9f5b01744bf438470f7